suites/regressions/queryPerformanceTest.js

```diff
@@ -0,0 +1,47 @@
+Feature('Query Performance Tests').tag('@regressions');
+
+/*
+* Before you use this test, generate the queries by running: 
+* TEST_DATA_PATH='' NAMESPACE=<your-namespace> npm test -- suites/regressions/generateQueries.js --verbose
+* to generate the queries which will be read in from the file.
+* 
+* Only then should you run this test file with:
+* TEST_DATA_PATH='' NAMESPACE=<your-namespace> npm test -- suites/regressions/queryPerformanceTest.js --verbose
+* 
+* This test generates and executes expensive GraphQL queries using a commmons' dictionary.json.
+* We assume a certain structure for the dictionary file, and the presence of a
+* top-level node called "program". Everything else should be portable between different commons. 
+*/
+
+var nodes = require('../../utils/nodes');
+var fs = require('fs');
+
+if (fs.existsSync('output/bottomUpQueries.txt')) {
```

Comment:
it would be great to use BeforeSuite to create all the data and use this to write `Scenario`
https://codecept.io/advanced

Reply:
could you open a user-story to handle the pattern suggested in https://codecept.io/advanced this later?

Reply:
I have PXP-2480 for this.